guides/micronaut-json-schema-generator/micronaut-json-schema-generator.adoc

```diff
@@ -0,0 +1,140 @@
+dependency:micronaut-json-schema-annotations[groupId=io.micronaut.jsonschema]
+
+The `sourcegen-generator` feature is used to generate Java code and adds the following dependencies:
+
+dependency:micronaut-sourcegen-generator-java[groupId=io.micronaut.sourcegen,scope=annotationProcessor]
+dependency:micronaut-sourcegen-annotations[groupId=io.micronaut.sourcegen]
+
+The `netty-server` feature is needed for the testing environment for our example client.
+
+== Generating the Java classes from JSON Schema
+:exclude-for-build:maven
+Add the Micronaut JsonSchema plugin to your plugins section:
+
+[source, groovy]
+.build.gradle
+----
+plugins {
+  id 'io.micronaut.jsonschema' version '4.5.0-SNAPSHOT'
+  ...
+}
+----
+
+And configure your build to generate from a JSON Schema URL:
+
+[source, groovy]
+.build.gradle
+----
+micronaut {
+  ...
+  jsonschema {
+    fromUrl("https://json.schemastore.org/openweather.current.json") {
+        outputPackageName.set("com.example.openweather")
+    }
+ }
+}
+----
+
+Micronaut JsonSchema generator generates the client code in your build directory and adds it as a source set.
+Therefore, you can, for example run `./gradlew compileJava --console=verbose` and see that the sources are generated and compiled:
+
+[source]
+----
+> Task :generatingSourcesFromOpenweather
+...
+> Task :compileJava
+----
+
+TIP: The Micronaut JsonSchema generator supports a variety of parameters. Please refer to https://micronaut-projects.github.io/micronaut-gradle-plugin/snapshot/#_source_generator_from_micronaut_json_schema[Micronaut JsonSchema Gradle plugin documentation] for all possible options.
+
+After generation finishes, you should see the following directory structure under your `target/generated-sources/jsonschema` directory:
+
+[source]
+----
+target/generated-sources/jsonschema
+└── generatingSourcesFromOpenweather
+    └── src
+        ├── main
+        │   └── java
+        │       └── com.example.openweather
+        │           └── OpenweatherCurrentWeatherDataApi.java
+----
+
+:exclude-for-build:
+
+:exclude-for-build:gradle
+
+Enable the Micronaut JsonSchema generation in your `pom.xml` file, and configure the location to the specification file and package names:
+
+[source, xml]
+.pom.xml
+----
+<properties>
+    <micronaut.jsonschema.generator.enabled>true</micronaut.jsonschema.generator.enabled>
+   <micronaut.jsonschema.generator.input-url>https://json.schemastore.org/openweather.current.json</micronaut.jsonschema.generator.input-url>
+<micronaut.jsonschema.generator.outputPackageName>com.example.openweather</micronaut.jsonschema.generator.outputPackageName>
+</properties>
+----
+Micronaut JsonSchema generator generates the client code in your target directory and compiles it automatically alongside your regular sources.
+Therefore, you can, for example run `mvn compile` and see that the sources are generated and compiled.
```

Comment:
Therefore, you can<s>, for example</s> run `mvn compile` and see that the sources are generated and compiled.

This would be easier to read I think